README.md

```diff
@@ -69,7 +69,8 @@ Extensions can override the following default extension points to add or change
     onEvent : function(name, evt) {return true;},
     transformResponse : function(text, xhr, elt) {return text;},
     isInlineSwap : function(swapStyle) {return false;},
-    handleSwap : function(swapStyle, target, fragment, settleInfo) {return false;},
+    handleSwap : function(swapStyle, target, fragment, settleInfo) {return [target];},
```

Comment:
Hey, see the lib's core:
https://github.com/bigskysoftware/htmx/blob/master/src/htmx.js#L4796

`handleSwap ` returns `false` by default, I think we should reuse the same default values in this README

Reply:
Good Catch! I copied the `extensionBase` object over 😉